style/wishlist.css

```diff
@@ -0,0 +1,29 @@
+.card__image > img {
+    width: 30rem;
+    height: 28rem;
+    object-fit: cover;
+}
+.card__buttons {
+    justify-content: center;
+}
+.card__buttons .btn {
+    width: 100%;
+}
+
+.main__container {
+    
+    border: 2px solid var(--gray);
+    border-radius: 1rem;
+    padding: 2rem;
+}
+.wishlist {
+    width: 100%;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+}
+@media only screen and (max-width: 640px) {
```

Comment:
You can use ems for media queries

Reply:
Thanks, I have changed the unit to em from pixel.